docs/data-collection.md

```diff
@@ -84,6 +84,24 @@ For the OpenShift 4 Developer Preview we will be sending back these exact attrib
   // subscription_sync_total is the number of times an OLM operator
   // Subscription has been synced, labelled by name and installed csv
   '{__name__="subscription_sync_total"}',
+  //
+  //OCS metrics to be collected:
+  // ceph_cluster_total_bytes gives the size of ceph cluster in bytes.
+  '{__name__="ceph_cluster_total_bytes"}',
+  // ceph_cluster_total_used_raw_bytes is the amount of ceph cluster storage used in bytes.
+  '{__name__="ceph_cluster_total_used_raw_bytes"}',
```

Comment:
Same naming note as above

Reply:
@squat This is not a recording rule. This is a metric directly collected from ceph. This has cardinality 1.

You can verify this in screenshot 2 of https://github.com/openshift/telemeter/pull/192#issuecomment-514568851